apps/cookbook/src/app/layout-recipes/layout-recipes.component.html

```diff
@@ -0,0 +1,55 @@
+<article>
+  <h1>Kirby Layout Recipes</h1>
+  <p>
+    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus dignissimos error quo,
+    deleniti voluptatum eveniet inventore pariatur a sapiente, quos magnam reprehenderit assumenda
+    eos rem cupiditate. Molestias libero optio deserunt.
```

Comment:
It this reserved for a small description, or will it be removed?

Reply:
I'll try to write a short description. Otherwise it will be removed :)